packages/manager/apps/pci-databases-analytics/src/pages/services/[serviceId]/indexPatterns/deletePattern/DeletePattern.modal.tsx

```diff
@@ -0,0 +1,105 @@
+import { useTranslation } from 'react-i18next';
```

Comment:
Same as `DeleteIndexModal` component, with `strictNullChecks` on, `service`, `deletedPattern` & `projectId` are possibly undefined and can lead to runtime errors